x/feeabs/ante/decorate.go

```diff
@@ -315,8 +315,8 @@ func (famfd FeeAbstrationMempoolFeeDecorator) AnteHandle(ctx sdk.Context, tx sdk
 		// Not contain zeroCoinFeesDenomReq's denoms
 		//
 		// check if the feeCoins has coins' amount higher/equal to nonZeroCoinFeesReq
-		if !feeCoins.IsAnyGTE(nonZeroCoinFeesReq) {
-			err := sdkerrors.Wrapf(errorstypes.ErrInsufficientFee, "insufficient fees; got: %s required: %s", feeCoins, feeRequired)
+		if !feeCoinsNonZeroDenom.IsAnyGTE(nonZeroCoinFeesReq) {
+			err := sdkerrors.Wrapf(errorstypes.ErrInsufficientFee, "insufficient fees; got: %s required: %s", feeCoinsNonZeroDenom, nonZeroCoinFeesReq)
```

Comment:
`feeCoins` is the initial fees(which might be an ibc denom), so when compare to `nonZeroCoinFeeReq`, which are the amount of fee required in native token, would fail if provided fee is IBC denom. `feeCoinsNonZeroDenom` is calculated from feeCoins and TwapPrices [here](https://github.com/osmosis-labs/fee-abstraction/blob/main/x/feeabs/ante/decorate.go#L284-L288), so should use it instead.